src/NuGetPackager/PackageCreator.cs

```diff
@@ -97,12 +97,12 @@ void CreatePackagesFromNuSpecFile(string nuSpec, string destinationFolderFullPat
                     packageBuilder.PopulateFiles("", manifest.Files);
             }
 
-            SavePackage(packageBuilder, destinationFolderFullPath, ".nupkg", "Package created -> {0}");
+            SavePackage(packageBuilder, destinationFolderFullPath, "nupkg", "Package created -> {0}");
```

Comment:
@bording As you already modified the value I suggest to just remove that extension argument? Not really useful.